corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -388,6 +388,21 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.templateType = 'select';
         self.choices = question.choices;
         self.isMulti = true;
+        self.hideLabel = ko.observable(options.hideLabel);
+
+        // A custom binding to add a column layout class only if hideLabel = true
+        ko.bindingHandlers.addColStyle = {
+            update: function(element, valueAccessor) {
+                var value = valueAccessor();
+                var shouldAddColStyle = ko.unwrap(value);
+
+                if (shouldAddColStyle) {
+                    // Account for number of choices plus column for clear button
```

Comment:
What would you think of DRYing this up with the other place this logic is used, maybe as a method on `ChoiceLabelEntry.prototype`?

Reply:
On it, you should see it added!

Reply:
Correct me if I'm wrong but I believe `Entry` is the common ancestor for `MultipleSelectEntry` and `ChoiceLabelEntry` so I added a col style prototype method there, let me know what you think!